package.json

```diff
@@ -63,6 +63,7 @@
     "bs58": "^3.0.0",
     "debug": "^2.2.0",
     "detect-node": "^2.0.3",
+    "fnv": "^0.1.3",
```

Comment:
Are we sure about the licence for this module, can't find info on npm nor the repo.